src/views-components/context-menu/action-sets/collection-action-set.ts

```diff
@@ -27,8 +28,8 @@ export const collectionActionSet: ContextMenuActionSet = [[
     {
         icon: MoveToIcon,
         name: "Move to",
-        execute: (dispatch, resource) => {
-            // add code
+        execute: (dispatch) => {
+            dispatch<any>(openMoveToDialog());
         }
```

Comment:
Can be shortened to 
`execute: dispatch => dispatch<any>(openMoveToDialog())` :)